spec/models/notification_spec.rb

```diff
@@ -272,24 +272,22 @@
     end
```

Comment:
I noticed how the logic of the specs has become more complicated, why did you take this approach?

Reply:
@rhymes thank you for your review comments.

`sidekiq_perform_enqueued_jobs` is only working in a `before` block. That's why I had to make things little complex. 

previously used expect{}.to change() is making sure that Sidekiq Jobs are executed.

For eg. this doesn't execute all enqueued Jobs before checking
```ruby
it "example" do
  sidekiq_perform_enqueued_jobs do 
    described_class.send_reaction_notification(reaction, reaction.reactable.user)
    expect {}.to change().by(1)
  end
end
```

If you can guide on this, I can make specs simple are previous.

Reply:
what if you do something like:

```ruby
expect do
  sidekiq_perform_enqueued_jobs do
    ... send reaction ...
  end
end.to change...

Reply:
Thank you @rhymes 
It worked like charm. I will update specs.